src/Razor/test/Microsoft.AspNetCore.Razor.VSCode.FunctionalTest/tests/TestUtil.ts

```diff
@@ -133,6 +145,16 @@ export async function waitForProjectConfigured(directory: string) {
     }, /* timeout */ 60000, /* pollInterval */ 250);
 }
 
+export async function restartOmnisharp() {
+    await vscode.commands.executeCommand('o.restart')
```

Comment:
Using await and `then` in the same method. Isn't that equivalent to just putting everything in the `then` at the end surrounded by a try catch to capture errors??

Reply:
Is it? From my understanding, this is how we handle both resolved and rejected promises. It's possible that the await here is unnecessary but we don't need any try catch.